pkg/kubelet/cm/cpumanager/state/state_mem.go

```diff
@@ -59,10 +59,10 @@ func (s *stateMemory) GetCPUSetOrDefault(containerID string) cpuset.CPUSet {
 	s.RLock()
```

Comment:
Can you instead just not lock here?

Reply:
Yes, that seems a better solution since atomic is not needed here.